client/src/components/Sidenav/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -8,87 +8,115 @@ exports[`<Sidenav /> should render component 1`] = `
       "maxWidth": "220px",
     }
   }
-  onHideNav={undefined}
+  onHideNav={[Function]}
   showNav={undefined}
```

Comment:
Now I see that maybe it would be great to create snapshots for both opened and closed navBar. Even though now the coverage is 100% it does not mean that I covered all 100% of the functionality.